src/models/hotspot.rs

```diff
@@ -42,6 +43,20 @@ pub struct Hotspot {
     pub speculative_nonce: u64,
     /// The current reward scale for the hotspot
     pub reward_scale: Option<f64>,
+    /// The current status for the hotspot
+    pub status: Status,
+}
+
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct Status {
+    /// The timestamp of hotspot status, when it last sends gossip
+    pub timestamp: DateTime<Utc>,
```

Comment:
Timestamp is an `Option<DateTime<Utc>>`

Reply:
Noted. Just curious, when would the API return null datetime such that we would need Option in this case?

Reply:
when the hotspot is offline is when we can see a status field with null values. For example

https://api.helim.io/v1/hotspots/13DTPjZeaJ8YKhrNyNuFdY8aCXHEpFuEUy7Qd2UDzLEQQqBi6sk

returns status as:

```
        "status": {
            "height": null,
            "listen_addrs": null,
            "online": "offline",
            "timestamp": null
        }
```

Reply:
I see, thank's for the clarification. I have made the change as needed.